sys/msdk/gstmsdkvp9enc.c

```diff
@@ -29,6 +29,22 @@
  * EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/**
+ * SECTION: element-msdkvp9enc
+ * @title: msdkvp9enc
+ * @short_description: Intel MSDK VP9 encoder
+ *
+ * VP9 video encoder based on Intel Media SDK
+ *
+ * ## Example launch line
+ * ```
+ * gst-launch-1.0 videotestsrc num-buffers=90 ! msdkvp9enc ! matroskamux ! filesink location=output.webm
+ * ```
+ *
+ * Since: 1.20
```

Comment:
Why is it 1.20?

Reply:
The next major release is 1.20.

Reply:
Yes, the next release is 1.20, however it seems you added descriptions for msdkvp9enc which was added a few years ago.  Here it will mislead user that msdkvp9enc is available from 1.20

Reply:
Yes, I noticed this problem before. It seems that “Since: 1.20” here is specially for the hotdoc, and the “Copyright (c) 2019 Intel Corporation” at the top of the cpp file is for msdkvp9enc availability.I haven't found a better way.Do you have any better suggestions?

Reply:
'Since: xxx' is used to indicate which version of gstreamer the new API / element will be added into. msdkvp9enc was added before, so we can not use 'Since: 1.20'